packages/mdc-ripple/index.js

```diff
@@ -114,3 +114,12 @@ export class MDCRipple extends MDCComponent {
     this.unbounded = 'mdcRippleIsUnbounded' in this.root_.dataset;
   }
 }
+
+/**
+ * @typedef {!{
+ *   root_: !Element,
+ *   unbounded: (boolean|undefined),
+ *   disabled: (boolean|undefined)
+ * }}
+ */
+let RippleAbleType;
```

Comment:
Done. I changed from `@typedef` to `@record` so that those comments can be inlined.